.github/workflows/main.yml

```diff
@@ -26,18 +26,22 @@ jobs:
       fail-fast: false
       matrix:
         # As of Jackson 2.14 got Java 8 baseline so can build on later JDKs too
-        java_version: ['8', '11', '17']
-        os: ['ubuntu-20.04']
+        java_version: ["8", "11", "17"]
```

Comment:
Any particular reason for this change?

Reply:
Oh sorry, that was my linter! You'll notice the last commit was fixing some whitespace changes it'd also done, but I missed this change, my apologies. I can undo it, if you prefer.